### PR TITLE
fix(types): fix continueWith flow types

### PR DESCRIPTION
**Pull request by briancavalier**
D'oh!

### CODE REVIEW

File: packages/core/src/index.js.flow [hunk 1]
```diff
@@ -62,8 +62,8 @@ declare export function mergeMapConcurrently <A, B> (concurrency: number): (f: (
 declare export function mergeMapConcurrently <A, B> (concurrency: number, f: (A) => Stream<B>): (s: Stream<A>) => Stream<B>
 declare export function mergeMapConcurrently <A, B> (concurrency: number): (f: (A) => Stream<B>) => (s: Stream<A>) => Stream<B>
 
-declare export function continueWith <A> (f: (any) => A, s: Stream<A>): Stream<A>
-declare export function continueWith <A> (f: (any) => A): (s: Stream<A>) => Stream<A>
+declare export function continueWith <A> (f: (any) => String<A>, s: Stream<A>): Stream<A>
```
Comment (TrySound) [inline packages/core/src/index.js.flow]: Are you sure you want `string` here?

Comment (Frikki) [reply packages/core/src/index.js.flow]: Yea, what is this?

Comment (briancavalier) [reply packages/core/src/index.js.flow]: Lol. Just making sure you're paying attention 😂

Comment (briancavalier) [reply packages/core/src/index.js.flow]: Fixed in 0f4998e

**Pull request merged**